web-model/src/main/java/com/oasis/webmodel/request/LoginRequestModel.java

```diff
@@ -0,0 +1,9 @@
+package com.oasis.webmodel.request;
+
+import lombok.Getter;
+
+@Getter
+public class LoginRequestModel {
```

Comment:
no need suffix "Model"
LoginRequest is enough

Reply:
Commit ID: 279a1f3